README.md

```diff
@@ -1 +1,124 @@
-# morse-code
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Decoder Morse messages. ](#-decode-morse-code-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Decoder Morse messages. <a name="about-project"></a>
+
+**Decoder Morse messages.**  A simple morse code decoder. Uisng Ruby language
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
```

Comment:
- [ ] Kindly fix your readme file. The other sections in the readme are hidden with this dropdown.

  ![image](https://user-images.githubusercontent.com/81468675/230336194-63d5e373-a271-45f8-b651-631fb7f263d6.png)